src/masternodes/rpc_proposals.cpp

```diff
@@ -457,8 +459,10 @@ UniValue votegov(const JSONRPCRequest &request) {
         vote = CProposalVoteType::VoteNo;
     } else if (voteStr == "yes") {
         vote = CProposalVoteType::VoteYes;
-    } else if (voteStr != "neutral") {
+    } else if (gArgs.GetBoolArg("-rpc-governance-accept-neutral", DEFAULT_RPC_GOV_NEUTRAL) && voteStr != "neutral") {
```

Comment:
Minor tweak: Let's just read the gArgs once upto top and make the casing a bit simpler?